test/OpenTelemetry.AutoInstrumentation.Loader.Benchmark/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- This file prevents unintended imports of unrelated MSBuild files -->
+  <!-- Uncomment to include parent Directory.Build.props file -->
+  <!--<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />-->
```

Comment:
It will be great to put here at least and remove it from csproj file.

```xml
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
```

Reply:
Done.